cmd/river/rivercli/command.go

```diff
@@ -164,3 +175,9 @@ func openPgxV5DBPool(ctx context.Context, databaseURL string) (*pgxpool.Pool, er
 
 	return dbPool, nil
 }
+
+func pgEnvConfigured() bool {
+	// Consider these two vars the bare minimum for an env-based configuration.
+	// return os.Getenv("PGHOST") != "" && os.Getenv("PGDATABASE") != ""
+	return os.Getenv("PGDATABASE") != ""
+}
```

Comment:
Some commented code in here and a comment which only lines up with the commented variant (not the active one)

Reply:
Ah good catch. I changed my mind on this one. `PGHOST` defaults nicely to `localhost` so it seems okay to assume that `PGDATABASE` could be considered a bare minimum viable configuration.